src/emcpy/calcs/units.py

```diff
@@ -1,36 +1,102 @@
+__all__ = ['K_to_C', 'K_to_F', 'C_to_K', 'C_to_F', 'F_to_C', 'mps_to_MPH',
+           'mm_to_inches']
+
 # Temperature
 def K_to_C(K):
-    """Convert Kelvin to Celsius"""
+    """
+    K_to_C : Convert Kelvin to Celsius
+
+    K_to_C(K)
+
+    INPUT:
+      K - temperature in kelvin
+    OUTPUT:
+      C - temperature in degrees celsius
```

Comment:
Ok. So we need to converge on a nomenclature.
See `io.netCDF` docstrings.

I am fine with whichever way we go, we just have to be consistent.

Reply:
Ok I don't have a strong opinion on which way to go either, as the `stats.py` file you added has different templates for it. @kevindougherty-noaa any preference on a way to go?

Reply:
Based on my research on Friday, I think the numpy/scipy docstrings is the best way to go. It is a combination of reStructured text which is 'feature rich' and Google's docstring which I think is cleaner. See attached link for info on each: https://realpython.com/documenting-python-code/#docstring-formats

Reply:
Works for me, I also think INPUT and OUTPUT scream FORTRAN, so I think this is a good choice.

Reply:
works for me too.